guides/common/modules/proc_deploying-a-custom-ssl-certificate-to-satellite-server.adoc

```diff
@@ -75,10 +75,27 @@ To update the certificates on a currently running Katello installation, run:
 endif::[]
 +
 Note that you must not access or modify `/root/ssl-build`.
-. From the output of the `katello-certs-check` command, depending on your requirements, enter the `{foreman-installer}` command that installs a new {Project} with custom SSL certificates or updates certificates on a currently running {Project}.
+. Run the command to update certificates on your {ProjectServer}:
 +
-If you are unsure which command to run, you can verify that {Project} is installed by checking if the file `/etc/foreman-installer/scenarios.d/.installed` exists.
-If the file exists, run the second `{foreman-installer}` command that updates certificates.
+ifdef::satellite[]
+[options="nowrap", subs="+quotes,attributes"]
+----
+# {foreman-installer} --scenario satellite \
```

Comment:
If you have an existing installation, the `--scenario` option is not needed because the installer will remember the last used scenario. As you stated earlier, we expect the user to have an existing installation.

Taking a step back and looking at the whole procedure, I'd suggest to drastically simplify it. The installer has [a hook](https://github.com/theforeman/foreman-installer/blob/develop/hooks/pre_commit/20-certs_update.rb) that runs `katello-certs-check` itself, so we can actually drop step 1 altogether.

At that point, the instructions won't need the `ifdef` at all and overall is much shorter.

As for cherry picking, https://projects.theforeman.org/issues/36567 was a significant bug so I'd take 3.6 as the lowest version that's safe. After https://github.com/theforeman/foreman-documentation/pull/3438 is merged, that shouldn't be a problem then.

That's perhaps a change that should be made before this in its own PR.

Reply:
Ok, thanks! Let's try to look at this in https://github.com/theforeman/foreman-documentation/pull/3486